backend-project/small_eod/events/models.py

```diff
@@ -9,12 +9,12 @@ class Event(TimestampUserLogModel):
     name = models.CharField(
         max_length=256, verbose_name=_("Name"), help_text=_("Name of event.")
     )
-    comment = models.CharField(
-        max_length=256, verbose_name=_("Comment"), help_text=_("Comment text.")
+    comments = models.CharField(
+        max_length=256, verbose_name=_("Comments"), help_text=_("Comments text.")
     )
-    case = models.ForeignKey(
+    cases = models.ForeignKey(
```

Comment:
@magul , czy też masz wrażenie, że w tym miejscu liczba mnoga jest nadmierna?

Reply:
Dobra, dotarłem do tego komentarza, ogarniemy to jakoś w środę.